### PR TITLE
update postcoder to the arm64 friendly version

### PR DESCRIPTION
**Pull request by danieldevlewis**
Version 4 is (somewhat) new, and removes a bunch of deprecated code.

Version 4.0.2 is the patch version that actually works in a container on an arm64 mac.

### CODE REVIEW

File: index.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ entries:
   elasticsearch:
   - apiVersion: v1
     appVersion: 7.6.2
-    created: "2022-07-28T15:53:49.115469+01:00"
+    created: "2022-08-30T17:17:06.97692+01:00"
```
Comment (danieldevlewis) [inline index.yaml:6]: Should these have updated?

Comment (michelesr) [reply index.yaml:6]: yes, they update automatically :-)

**Review by michelesr — APPROVED** [review]
LGTM

**Comment by michelesr** [comment]
@MatthewCane can we merge this now or we have to wait for your PR to go in?

**Review by MatthewCane — APPROVED** [review]

**Pull request merged**